### PR TITLE
76: Play Documentation

### LINKED ISSUE #76: PRX Play Documentation
Documentation Copy: https://docs.google.com/document/d/12COnYedPOAAzbzftyUe-Gio6HweV_F-wVEZjnY4vRhc/edit#heading=h.lmfg5xiozyp1
Design: https://www.figma.com/file/JX0HPXGk5xt8s1NWZBXjgT/PRX-Play?node-id=33%3A736

Update homepage with documentation based on designed layout.

### PR DESCRIPTION
**Pull request by a-maci29**
Closes #76

- Adds documentation to the home page

## To Review

- [ ] Checkout Branch.
- [ ] Run `asdf install`.
- [ ] Run `yarn`.
- [ ] Run `yarn dev`.
- [ ] Go to [localhost:4300](http://localhost:4300).

> ...then...

- [ ] Review documentation and styling
- [ ] Ensure page is responding as expected for screen changes
- [ ] Ensure all links are functional, target correct section

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,25 +1,441 @@
...
+          <h3>
+            <a href="#customization-parameters" id="customization-parameters">
+              Customization Parameters
+            </a>
+          </h3>
+          <p>
+            These parameters are supported by both the RSS Feed and Audio File
+            embed players. All parameters are optional.
+          </p>
+          <h4>ui</h4>
```
Comment (kookster) [inline pages/index.tsx:274]: Should this be `ui` or `ue`? the example below is `ue`

**Review by kookster — CHANGES_REQUESTED** [review]
A confusion on `ue` or `ui` that should get fixed.
Some possible content suggestions.

File: pages/index.tsx (hunk 1 above)
Comment (rpeterman-gp) [reply pages/index.tsx:274]: The example is incorrect. It should use the `ui` parameter.

**Review by kookster — APPROVED** [review]
lgtm!

**Pull request merged**